ci-operator/jobs/kubevirt/kubevirt/kubevirt-kubevirt-master-presubmits.yaml

```diff
@@ -0,0 +1,185 @@
+presubmits:
+  kubevirt/kubevirt:
+  - agent: kubernetes
+    always_run: true
+    branches:
+    - master
+    context: ci/prow/dummy-test
+    decorate: true
+    decoration_config:
+      skip_cloning: true
+    labels:
+      ci-operator.openshift.io/prowgen-controlled: "true"
+      pj-rehearse.openshift.io/can-be-rehearsed: "true"
+    name: pull-ci-kubevirt-kubevirt-master-dummy-test
+    path_alias: kubevirt.io/kubevirt
+    rerun_command: /test dummy-test
+    spec:
+      containers:
+      - args:
+        - --artifact-dir=$(ARTIFACTS)
+        - --branch=master
+        - --give-pr-author-access-to-namespace=true
+        - --org=kubevirt
+        - --repo=kubevirt
+        - --resolver-address=http://ci-operator-configresolver-ci.svc.ci.openshift.org
+        - --sentry-dsn-path=/etc/sentry-dsn/ci-operator
+        - --target=dummy-test
+        command:
+        - ci-operator
+        env:
+        - name: CONFIG_SPEC
+          valueFrom:
+            configMapKeyRef:
+              key: kubevirt-kubevirt-master.yaml
+              name: ci-operator-master-configs
+        image: ci-operator:latest
+        imagePullPolicy: Always
+        name: ""
+        resources:
+          requests:
+            cpu: 10m
+        volumeMounts:
+        - mountPath: /etc/sentry-dsn
+          name: sentry-dsn
+          readOnly: true
+      serviceAccountName: ci-operator
+      volumes:
+      - name: sentry-dsn
+        secret:
+          secretName: sentry-dsn
+    trigger: (?m)^/test( | .* )dummy-test,?($|\s.*)
+  - agent: kubernetes
+    always_run: true
+    branches:
+    - master
+    context: ci/prow/kubevirt-e2e-test
+    decorate: true
+    decoration_config:
+      skip_cloning: true
+    labels:
+      pj-rehearse.openshift.io/can-be-rehearsed: "true"
+    name: pull-ci-kubevirt-kubevirt-master-kubevirt-e2e-test
+    path_alias: kubevirt.io/kubevirt
+    rerun_command: /test kubevirt-e2e-test
+    spec:
+      containers:
+      - args:
+        - --artifact-dir=$(ARTIFACTS)
+        - --branch=master
+        - --give-pr-author-access-to-namespace=true
+        - --lease-server=http://boskos
+        - --org=kubevirt
+        - --repo=kubevirt
+        - --resolver-address=http://ci-operator-configresolver-ci.svc.ci.openshift.org
+        - --secret-dir=/usr/local/kubevirt-e2e-test-cluster-profile
+        - --sentry-dsn-path=/etc/sentry-dsn/ci-operator
+        - --target=kubevirt-e2e-test
+        - --template=/usr/local/kubevirt-e2e-test
+        command:
+        - ci-operator
+        env:
+        - name: CLUSTER_TYPE
+          value: gcp
+        - name: CONFIG_SPEC
+          valueFrom:
+            configMapKeyRef:
+              key: kubevirt-kubevirt-master.yaml
+              name: ci-operator-master-configs
+        - name: JOB_NAME_SAFE
+          value: kubevirt-e2e-test
+        - name: TEST_COMMAND
```

Comment:
All of those should be inside the template and you should add here only your test commands. 

Also, you need to do some changes in `prowgen` as well to include the new template. see [here](https://github.com/openshift/ci-tools/blob/master/cmd/ci-operator-prowgen/main.go#L300-L347) and [here](https://github.com/openshift/ci-tools/blob/master/pkg/api/config.go#L262-L329)

Reply:
We've created a new template based on `cluster-launch-installer-libvirt-e2e.yaml`, I dont see any reference for that one? Does that mean that for that template the changes are pending or aren't they required as this is too special?